zebra-state/src/service.rs

```diff
@@ -530,6 +530,13 @@ impl Service<Request> for StateService {
     type Future =
         Pin<Box<dyn Future<Output = Result<Self::Response, Self::Error>> + Send + 'static>>;
 
+    // ## Correctness:
```

Comment:
Delete this, it doesn't apply, because buffers can return pending when they get full.